### DISCUSSION TITLE
DSP or Convolution integration?

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#1354] [Ideas]
Based to this https://github.com/MusicPlayerDaemon/MPD/discussions/2127#discussioncomment-10855264

Is DSP or a Convolution software(better option?) integration possible to myMPD? 
I know some people use these features to their environment..

**Comment by jcorporation** [comment]
I do not use such software. In my MPD setup are the outputs:

- Alsa Hardware device 
- Http Stream
- Bluetooth

**Comment by Emilot** [comment]
I am not either but i know some friends of mine and i see in social media that a lot are using them.

You use Bluetooth in what way? I hope not to transfer audio....

**Reply by jcorporation** [reply]
With Bluealsa to my Bluetooth speakers.

**Reply by Emilot** [reply]
> With Bluealsa to my Bluetooth speakers.

Everytime i tried to use a wireless connection always go back to wired, in terms of audio quality.

**Reply by jcorporation** [reply]
In this setup I listen mostly webradios and the audio quality is good enough for this.

If I want to listen high quality audio I use my rega record player.

I prefer the sound of analog audio.

**Reply by Emilot** [reply]
> In this setup I listen mostly webradios and the audio quality is good enough for this.
> 
> If I want to listen high quality audio I use my rega record player.
> 
> I prefer the sound of analog audio.

Same taste for analog audio(that is why i choose Sabre for my digital equipment). (Lenco L78 for vinyl)
